Index: httplib2/__init__.py
===================================================================
--- httplib2/__init__.py        (revision 16732)
+++ httplib2/__init__.py        (working copy)
@@ -852,6 +852,7 @@
     def _conn_request(self, conn, request_uri, method, body, headers):
         for i in range(2):
             try:
+                conn.connect()
                 conn.request(method, request_uri, body, headers)
             except socket.gaierror:
                 conn.close()
